pylmod/base.py

```diff
@@ -48,82 +45,84 @@ def __init__(
         self.cert = cert
 
         self.urlbase = urlbase
+        if not urlbase.endswith('/'):
+            self.urlbase += '/'
         self.ses = requests.Session()
         self.ses.cert = cert
         self.ses.timeout = self.TIMEOUT  # connection timeout
         self.ses.verify = True  # verify site certificate
+        # Mount the retry adapter to the base url
+        self.ses.mount(urlbase, HTTPAdapter(max_retries=self.RETRIES))
 
         log.debug("------------------------------------------------------")
-        log.info("[PyLmod] init base=%s", urlbase)
+        log.info("[PyLmod] init urlbase=%s", urlbase)
+
+    @staticmethod
+    def _data_to_json(data):
+        """Convert to json if it isn't already a string"""
+        if type(data) not in [str, unicode]:
```

Comment:
Question: this function ensures that if the data isn't already a string or unicode, it is converted to json.  The function name suggests that it simply does a json conversion.  If I want json, why would I settle for a string or unicode that isn't necessarily in json format?